target_csv.py

```diff
@@ -61,8 +61,7 @@ def persist_messages(delimiter, quotechar, messages, destination_path, fixed_hea
 
             validators[o['stream']].validate(o['record'])
 
-            filename = o['stream'] + '-' + now + '.csv'
-            filename = os.path.expanduser(os.path.join(destination_path, filename))
+            filename = generate_filename(filename_include_date, destination_path, o['stream'], ".csv", now)
```

Comment:
This won't work when filename_include_date is False. Pretty simple fix but I"ll close the PR for now as this doesn't work for everyone